bin/main.rb

```diff
@@ -1,37 +1,52 @@
 #!/usr/bin/env ruby
 require '../lib/engine.rb'
```

Comment:
[OPTIONAL]
It is convention to use `require_relative` for file in the directory of the project and `require` for external files like gems etc.